.github/workflows/database.yml

```diff
@@ -67,7 +67,7 @@ jobs:
             test: true
           - os: macOS
             device: localhost
-            test: true
+            test: false
```

Comment:
Please add a comment that the test is flaky and anything else you can say about the flake